web/views_whiteboard.py

```
@@ -0,0 +1,222 @@
+        whiteboard.save()
+
+        return JsonResponse({
+            'success': True,
+            'message': 'Whiteboard saved successfully'
+        })
+
+    except json.JSONDecodeError:
+        return JsonResponse({
+            'error': 'Invalid request data', 
+        }, status=400) 
+    except Exception: 
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)